CHANGELOG.md

```diff
@@ -9,15 +9,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- ### Added -->
 
-<!-- ### Changed -->
+### Changed
+
+- [Auth] JWT now expects proper format -- `Bearer ...` -- [rfc6749](https://tools.ietf.org/html/rfc6749#section-7.1).
 
 - [Proxy] When forwarding requests, RIG related meta data (e.g. correlation ID) in CloudEvents is now put into an object under the top-level key "rig". Note that in terms of the current [CloudEvents 0.2](https://github.com/cloudevents/spec/blob/v0.2/spec.md) specification this makes "rig" an [extension](https://github.com/cloudevents/spec/blob/v0.2/primer.md#cloudevent-attribute-extensions). Also, all RIG related keys have been renamed from snake_case to camelCase.
 
 <!-- ### Deprecated -->
 
 <!-- ### Removed -->
 
-<!-- ### Fixed -->
+### Fixed
+
+- [Inbound] Wrong pattern match for CORS handling when sending proxy request to Kafka/Kinesis. Before -- `%{"method" => "OPTIONS"}`, after -- `%{method: "OPTIONS"}`.
```

Comment:
Ah I didn't quite get what this was about. Do you think it makes sense to put implementation details into the Changelog? Assuming this file is targetted at end users, I guess something like this might suffice:

`- [Inbound] Fixed response to CORS related preflight request.`